modules/ROOT/partials/configuration/sidebar_show.adoc

```diff
@@ -0,0 +1,24 @@
+[[sidebar_show]]
+== `+sidebar_show+`
+This option allows you to show the specified sidebar on initialization.
```

Comment:
We probably need an admonition indicating this only available in TinyMCE 6.1 and later

Reply:
I added the admonition